pkg/ocm/output/nodepools.go

```diff
@@ -134,3 +135,15 @@ func PrintNodePoolManagementUpgrade(upgrade *cmv1.NodePoolManagementUpgrade) str
 
 	return ""
 }
+
+func PrintNodePoolDiskSize(aws *cmv1.AWSNodePool) string {
+	diskSizeStr := "default"
```

Comment:
why not use the number of default value, but use "default" string here?

Reply:
I suppose in case default changes internally. Even if the default value is returned in the API it will change it to whatever the number is. It is just in case some record does not have the value set yet I suppose

Reply:
yeah I think @gdbranco is right, I also just want the default to be the same as it already is for classic machinepools. the "default" string should never appear anyway once the API changes are merged since that value will always be set